source/images/content-header/download-reverse-full.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="44px" height="44px" viewBox="0 0 44 44">
+
+    <title xml:lang="en">Download</title>
+
+    <defs>
+
+        <style>
+
+            .black {
+                color: #212121;
+            }
+
+            .white {
+                color: #ffffff;
+            }
+
+            .arrow {
+                fill: currentColor;
+                stroke: none;
+            }
+
+            .circle {
+                fill: none;
+                stroke: currentColor;
+                stroke-width: 2;
+            }
+
+        </style>
+
+    </defs>
+
+    <g id="icon" class="white">
```

Comment:
There should be a way to use SVG fragment identifiers to just have a single image.

Reply:
Had a go, got it working before compression. Inline styles caused `currentColor` to not work.

Will leave for now and follow up.